docs/csharp/language-reference/keywords/switch.md

```diff
@@ -7,6 +7,8 @@ f1_keywords:
   - "switch"
   - "case"
   - "case_CSharpKeyword"
+  - "defaultcase_CSharpKeyword"
+  - "defaultcase"
```

Comment:
This looks redundant. Only `defaultcase_CSharpKeyword` is needed based on your Roslyn PR.

Reply:
Add, I wasn't aware, I kept seeing `foo` and `foo_CSharpKeyword`, so I thought both were necessary.  Thanks for letting me know.